ERCS/erc-7653.md

```diff
@@ -0,0 +1,96 @@
+---
+eip: 7653
+title: Named NFT
+description: An extension to ERC-721 based on ERC-7632.
```

Comment:
Your description doesn't really say anything about what's in this proposal. It only says where to look for more information.

It'd be nice to see at a glance what's actually in this document.